locales/en/index.yml

```diff
@@ -558,7 +558,7 @@ authentication:
         hint: Carousel that contains an introduction guide to IO functionalities. Select to know the content.
     session_expired:
       title: Your session is expired
-      body: Per ragioni di sicurezza, ti chiediamo di entrare con SPID o CIE ogni {{days}} giorni o se esci dall’app.
+      body: For security reasons, we ask you to log in with SPID or CIE every {{days}} day or if you log out
```

Comment:
We need to start using [plurals](https://www.i18next.com/translation-function/plurals) do differentiate 1 day from 5 day**s**.
Take note.

Reply:
Thank you! In this case days are always plural and we have a translation issue. We need to update `day` with `days`.

Reply:
Fixed! Thank you! 4e6cc4f29a8acd59d0b6627f45137bc3a664d4c4